### ISSUE TITLE
Version calculation includes package-name as prefix as of v3.2.4

### ISSUE THREAD

**Issue by tshowers-bt** [google-github-actions/release-please-action#491]
### TL;DR

Existing projects are having their versions reset to 1.0.0 because the `package-name` is being included as a prefix when searching for the latest released version. I've left [a comment](https://github.com/googleapis/release-please/commit/628a562a21245ed02546aa0c40c97a0e3d50a0c7#r73961429) on what I believe to be the offending code change.

### Expected behavior

Projects whose workflows have not changed should continue to continue to calculate versions as they have with previous `v3` versions.

### Observed behavior

Projects are including `package-name` as a prefix when searching for the latest release version and therefore determining there have been no releases.

### Action YAML

```yaml
name: "Release Please Build"

on:
  push:
    branches: [master]

env:
  PACKAGE_NAME: Eventing.Kafka

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - name: Setup Release Please
        uses: google-github-actions/release-please-action@v3
        id: release
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          release-type: simple
          package-name: ${{ env.PACKAGE_NAME }}
```

**Comment by chingor13** [comment]
Please try with the latest upgraded version which should have fixed this.

**Comment by AllexVeldman** [comment]
Ran into the same thing with v3.2.5, removing the `package-name` made sure the branch created and the tag lookup was correct.

**Comment by ChristianTellefsen** [comment]
I tried removing package-name just now, but it still updated the wrong branch. Retrying using 3.2.3...

**Comment by ChristianTellefsen** [comment]
Using 3.2.3 works fine for me.

**Comment by chingor13** [comment]
The changes in the latest versions of release-please were around the branch name used. `package-name` should only be necessary to configure if you are using `monorepo-tags` (component prefix in the release tag).

@tshowers-bt Is your scenario working with the latest v3 (3.2.5)?

@ChristianTellefsen Can you add your config and the branch name of the last merged release pull request?

**Comment by tshowers-bt** [comment]
[chingor13](https://github.com/chingor13) I've pinned to 3.2.3 (everything works fine there), I'll try 3.2.5 shortly and let you know. 

As a general recommendation should `package-name` be omitted for single-package repos? We have dozens of services/libraries and tend to use the same basic workflow so I've been adding it entirely because it was there in my "working examples". I'd call this a breaking change either way and hope 3.2.5 works without any updates to the workflows required, but in anticipation of an eventual v4 I'd remove the `package-name` anyway.

**Comment by ChristianTellefsen** [comment]
> @ChristianTellefsen Can you add your config and the branch name of the last merged release pull request?

Which config do you mean?

**Comment by ChristianTellefsen** [comment]
We got an issue in one of our packages after all this.  Not sure what is going on. As far as I can tell, the package version in package,json is not the same version as any released package. I cannot see from the logs which release it is actually trying to push.

```
12s
Run google-github-actions/release-please-action@v3.2.3
✔ Looking for latest release on branch: main with prefix: 
✔ Building releases
✔ Building strategies by path
✔ Building release for path: .
⚠ Duplicate release tag: tagName
Error: release-please failed: Validation Failed: {"resource":"Release","code":"already_exists","field":"tag_name"}
```

**Comment by tshowers-bt** [comment]
I seem to be getting the expected version with the latest v3 though [the action output](https://github.com/tshowers-bt/scratch/runs/6615039844?check_suite_focus=true) is still referring to looking for the latest release with a specific prefix.

![image](https://user-images.githubusercontent.com/68649564/170560111-f71960be-59fa-4a8c-a500-f0b0b3b20d72.png)

where [previous action runs](https://github.com/tshowers-bt/scratch/runs/5425769739?check_suite_focus=true) did not.

![image](https://user-images.githubusercontent.com/68649564/170560244-c4b58678-1cf2-4fcd-9faf-f2816629802c.png)

I have no stake in the log message but it seems odd for the bug to leave a vestige like that

**Comment by chingor13** [comment]
Closing as stale
